packages/terra-core-docs/src/terra-dev-site/doc/icon/Themeable.3.doc.mdx

```diff
@@ -0,0 +1,19 @@
+import { Badge } from 'terra-icon/package.json?dev-site-package';
```

Comment:
I've never seen this import syntax before. Interesting...

Reply:
I also hadn't. Copied it over from the existing doc pages. Still not 100% sure what it is and searching hasn't really helped.
Any ideas?